src/angular/styles/_typography.scss

```diff
@@ -800,6 +802,8 @@
       --sbb-menu-item-label-active: var(--sbb-color-white);
       --sbb-menu-item-background-color-hover: transparent;
       --sbb-menu-item-background-color-active: var(--sbb-color-charcoal);
+      --sbb-menu-panel-divider-color: var(--sbb-color-graphite);
```

Comment:
Try styling the panel globally. If it looks different across components in Figma, it's probably not intended. We'll later ask Manuel.